nvflare/private/fed/client/training_cmds.py

```diff
@@ -124,7 +125,8 @@ def process(self, req: Message, app_ctx) -> Message:
         if secure_train and not from_hub_site:
             workspace = Workspace(root_dir=engine.args.workspace, site_name=client_name)
             app_path = workspace.get_app_dir(job_id)
-            if not verify_folder_signature(app_path):
+            root_ca_path = os.path.join(workspace.get_startup_kit_dir(), "rootCA.pem")
```

Comment:
not related to this PR,
For these "rootCA.pem", "server.crt", "client.crt" maybe we can define in a file these names as constants?

Reply:
Thanks.  Will do that in the future PR.